src/AltinnCore/Designer/Controllers/DeployController.cs

```diff
@@ -93,6 +100,51 @@ public async Task<IActionResult> StartDeployment(string org, string service)
                 });
             }
 
+            // register application in platform storage
+            try
+            {
+                using (HttpClient client = new HttpClient())
+                {
+                    string applicationId = $"{org}-{service}";
+                    string versionId = $"{masterBranch.Commit.Id}";
+
+                    string storageEndpoint = Environment.GetEnvironmentVariable("PlatformStorage__ApiEndPoint") ?? _storage_settings.ApiEndPoint;
+                    ApplicationMetadataClient applicationMetadataClient = new ApplicationMetadataClient(client, storageEndpoint);
+
+                    ApplicationMetadata application = null;
+                    string message;
+
+                    try
+                    {                         
+                        application = applicationMetadataClient.GetApplicationMetadata(applicationId);
+                        message = $"updated from versionId {application.VersionId}";
+                    }
+                    catch (Exception)
```

Comment:
Any reason this exception is not logged like the other one?

Reply:
Yes, if the application doesn't exist GetApplicationMetadata will throw exception. Hence, the catch attempts to run createApplicationMetadata.